cli.py

```diff
@@ -0,0 +1,273 @@
+        pass
+    else:
+        if args.detformat == 'xywh':
+            # x,y,width, height
+            BB_FORMAT = BBFormat.XYWH
+        elif args.detformat == 'xyrb':
+            # x,y,right,bottom
+            BB_FORMAT = BBFormat.XYX2Y2
+        else:
+            raise Exception("%s is not a valid detection annotation format"%args.anno_det)
+    
+        if args.detcoord == 'abs':
+            COORD_TYPE = CoordinatesType.ABSOLUTE
+        elif args.detcoord == 'rel':
+            COORD_TYPE = CoordinatesType.RELATIVE
+        else:
+            raise Exception("%s is not a valid detection coordinate format"%args.detcoord)
+        det_anno = converter.text2bb(args.anno_det, bb_type=BBType.DETECTED, bb_format=BB_FORMAT, type_coordinates=COORD_TYPE, img_dir=args.img)
+
+        # If VOC specified, then switch id based to string for detection bbox:
+        #if args.gtformat == 'voc' or args.gtformat == 'imagenet':
+        if args.names != '':
+            # if names file not given, assume id-based detection output
+            with open(args.names, 'r') as r:
+                names = list(map(str.strip, r.readlines()))
+                for det in det_anno:
+                    _out = names[int(det._class_id)]
+                    det._class_id = _out
+
+    # print out results of annotations loaded:
+    print("%d ground truth bounding boxes retrieved"%(len(gt_anno)))
+    print("%d detection bounding boxes retrieved"%(len(det_anno)))
+
+    # compute bboxes with given metric:
+
+    # COCO (101-POINT INTERPOLATION)
+    if args.metrics == 'coco':
+        logging.info("Running metric with COCO metric")
+
+        # use coco_out for PR graphs and coco_sum for just the AP
+        coco_sum = coco_evaluator.get_coco_summary(gt_anno, det_anno)
+        coco_out = coco_evaluator.get_coco_metrics(gt_anno, det_anno, iou_threshold=args.threshold)
+        
+        value_only = tuple([float(_i[1]) for _i in coco_sum.items()])
+        print( ('\nCOCO metric:\n'
+                'AP [.5:.05:.95]: %f\n'
+                'AP50: %f\n'
+                'AP75: %f\n'
+                'AP Small: %f\n'
+                'AP Medium: %f\n'
+                'AP Large: %f\n'
+                'AR1: %f\n'
+                'AR10: %f\n'
+                'AR100: %f\n'
+                'AR Small: %f\n'
+                'AR Medium: %f\n'
+                'AR Large: %f\n\n'%value_only) )
+
+        print("Class APs:")
+        for item in coco_out.items():
+            if item[1]['AP'] != None:
+                print("%s AP50: %f\n"%(item[0], item[1]['AP']))
+            else:
+                logging.warning('AP for %s is None'%(item[0]))
+
+        if args.prgraph:
+            plot_coco_pr_graph(coco_out, mAP=coco_sum['AP50'], ap50=coco_sum['AP'], savePath=args.savepath, showGraphic=False)
+        return coco_sum
+
+    # 11-POINT INTERPOLATION:
+    elif args.metrics == 'voc2007':
+        logging.info("Running metric with VOC2007 metric")
```

Comment:
I'd include the approach name "Running metric with VOC2012 metric, using the 11-point interpolation approach" instead of simply "Running metric with VOC2007"

Reply:
Fixed in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/27394185d27d073b2f871dbd7e836ee038c16ad9